docs/Documentation/Applications/vasp.md

```diff
@@ -0,0 +1,537 @@
+
+To determine the electronic ground state, VASP makes use of efficient iterative matrix diagonalization techniques, like the residual minimization method with direct inversion of the iterative subspace (RMM-DIIS) or blocked Davidson algorithms. These are coupled to highly efficient Broyden and Pulay density mixing schemes to speed up the self-consistency cycle.
+
+For further details, documentation, forums, and FAQs, see the [VASP website](https://www.vasp.at/).
+
+## Accessing VASP on NREL's HPC Clusters
+
+The VASP license requires users to be a member of a "workgroup" defined by the University of Vienna or Materials Design. If you are receiving "Permission denied" errors when trying to use VASP, you must be made part of the "vasp" Linux group first. To join, please contact [HPC Help](mailto:hpc-help@nrel.gov) with the following information:
+
+- Your name
+- The workgroup PI
+- Whether you are licensed through Vienna (academic) or Materials Design, Inc. (commercial)
+- If licensed through Vienna:
+    - The e-mail address under which you are registered with Vienna as a workgroup member (this may not be the e-mail address you used to get an HPC account)
+    - Your VASP license ID
+- If licensed through Materials Design:
+    - Proof of current licensed status
+
+Once status can be confirmed, we can provide access to our VASP builds. 
+
+## Supported Versions
+
+NREL offers support for VASP 5 and VASP 6 on CPUs as well as GPU builds on certain systems. (See table below for details).
+
+- For CPUs, VASP can be built with either Intel compilers/Intel MPI or GNU compilers/Open MPI. In general the Intel MPI builds exhibit fastest performance and are recommended over the Open MPI builds.
+
+- For GPUs, VASP can be built with Cuda or with OpenACC. The OpenACC GPU-port of VASP was released with VASP 6.2.0, and the Cuda GPU-port of VASP was dropped in VASP 6.3.0. The OpenACC build shows significant performance improvement compared to the Cuda build, but is more susceptible to running out of memory. 
+
+|                    |     Eagle     |     Swift     |   Vermilion   |
+| ------------------ | ------------- | ------------- | ------------- |
+| VASP 5 (Intel MPI) |       x       |               |       x       |
+| VASP 6 (Intel MPI) |       x       |       x       |       x       |
+| VASP 6 (Open MPI)  |       x       |       x       |       x       |
+| GPU VASP (Cuda)    |       x       |               |       x       |
+| GPU VASP (OpenAcc) |       x       |               |       x       | 
+
+Three executables have been made available with each build of VASP:
+
+1. `vasp_gam` is for Gamma-point-only runs typical for large unit cells;
+
+2. `vasp_std` is for general k-point meshes with collinear spins; and,
```

Comment:
Should there be something after the and?

Reply:
No, it's item 3 after the "and". I think this is just punctuating items 1, 2, 3 as a sentence with the ";" and "and,"
Ok to remove if you prefer. I chose to leave them in.